### ISSUE TITLE
Re-enable axis scale options in context menu

### ISSUE THREAD

**Issue by bear24rw** [epezent/implot#502]
Would be great to get this feature re-enabled:

https://github.com/epezent/implot/blob/85573fe04ad27f912074a2e7cde889fdf912b0c6/implot.cpp#L1417-L1425

**Comment by epezent** [comment]
Yes, I'd like to re-enable this as well. I need to refresh my memory what exactly the problem was, but I know its existence was complicated by the introduction of `SetupAxisScale`.

**Comment by bear24rw** [comment]
Thanks. I'll try to procrastinate on this feature request from my users and hope you get to it first :)
